lib/foreman/cast.rb

```diff
@@ -0,0 +1,26 @@
+module Foreman
+  module Cast
+    def self.to_bool(value)
+      case value
+
+      when String
+        return true if value =~ (/\A(true|t|yes|y|on|1)\z/i)
+        return false if value.blank? || value =~ (/\A(false|f|no|n|off|0)\z/i)
```

Comment:
Because an empty string and null evaluate to false in most languages.  Apparently in ruby (always the rebel!) an empty string evaluates as true so I can see the argument to stay consistent with the language even if it makes the method less intuitive.  Since `nil` evaluates as false I think converting it to false makes sense as this is a `to_bool` method.

That being said I did a quick poll in #theforeman-dev and the unanimous decision was false for both.